packages/edit-site/src/components/page-content-focus-manager/edit-template-notification.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect, useDispatch } from '@wordpress/data';
+import { useEffect, useRef } from '@wordpress/element';
+import { store as noticesStore } from '@wordpress/notices';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { store as editSiteStore } from '../../store';
+
+/**
+ * Component that displays a 'Edit your template to edit this block'
+ * notification when the user is focusing on editing page content and clicks on
+ * a disabled template block.
+ *
+ * @param {Object}                                 props
+ * @param {import('react').RefObject<HTMLElement>} props.contentRef Ref to the block
+ *                                                                  editor iframe canvas.
+ */
```

Comment:
```suggestion
 * @return {null} Returns null.
 */
```

Reply:
The return type is `void`, which is what JSDoc assumes by default, no?

Reply:
Oh, no worries. I was thinking in Typescript where `void` is a proper type, meaning a function returns `undefined`.

I also assumed that, because this function returns `null`, it represents a missing object.

Reply:
OH I'm looking at the wrong function. You're right. Senior moment.

Reply:
Nit: this component now returns a component, should it have a return type of ` * @return {WPComponent} The component to be rendered.` added in the JSDoc?